BUILD.md

```diff
@@ -40,6 +41,39 @@ The `npm run watch` script in action:
 
 ## Getting Started
 
+### Quickstart with VSC Devcontainer for Hyperledger Cactus
+
+* Opening the Repo Within a Docker Container
+  * Install Prerequisites:
+    * Visual Studio Code (VSC)
+    * Docker Desktop (ensure it's running) https://docs.docker.com/engine/install/
+    * The Remote - Containers VSC extension
+  * Clone the Repository:
+    ```
+    git clone https://github.com/hyperledger-cacti/cacti.git
+    cd cacti
+    ```
+  * Open in VSC and Reopen in Container:
+    * Launch VSC
+    * Open the cacti folder
+    * Press Ctrl+Shift+P → Select Reopen in Container.
```

Comment:
Instead use this `Press F1 and type "Dev Containers: Open Folder in Container.."`